client/components/ManageTestQueue/index.jsx

```diff
@@ -675,13 +1090,241 @@ const ManageTestQueue = ({
+                                                    </PhasePill>{' '}
+                                                </td>
+                                                <td>{at.name}</td>
+                                                <td>{browser.name}</td>
+                                                <td>
+                                                    <TransparentButton>
+                                                        <FontAwesomeIcon
+                                                            icon={faEdit}
+                                                            color="#818F98"
+                                                            onClick={() => {
+                                                                setIsDelete(
+                                                                    false
+                                                                );
+                                                                setActionButtonLabel(
+                                                                    'Save Changes'
+                                                                );
+                                                                setUpdateAtIdForUpdate(
+                                                                    at.id
+                                                                );
+                                                                setUpdateBrowserIdForUpdate(
+                                                                    browser.id
+                                                                );
+                                                                setUpdatePhaseForUpdate(
+                                                                    phase
+                                                                );
+                                                                onOpenShowEditAtBrowserModal(
+                                                                    'edit',
+                                                                    phase
+                                                                );
+                                                            }}
+                                                        />
+                                                        <span className="sr-only">
+                                                            Edit
+                                                        </span>
+                                                    </TransparentButton>
+                                                    <TransparentButton>
+                                                        <FontAwesomeIcon
+                                                            icon={faTrashAlt}
+                                                            color="#ce1b4c"
+                                                            onClick={() => {
```

Comment:
`onClick` is on icon instead of the `button`. This makes the button inaccessible to keyboard users.

Reply:
Hey nice catch! I just moved the `onClick()` to the `<TransparentButton />` component.